rpki-validator/src/main/java/net/ripe/rpki/validator3/util/TrustAnchorLocator.java

```diff
@@ -172,14 +172,6 @@ public String getPublicKeyInfo() {
         return prefetchUris;
     }
 
-    @Override
-    public String toString() {
```

Comment:
I hope this is only for logging (nobody consume or use it for other purpose). Unless lombok.ToString produces same output?

Reply:
I think it is only used for debugging and the lombok to string should produce similar output.

Reply:
I had a  breakpoint there and did not hit it.